examples/common_vmseries_and_autoscale/example.tfvars

```diff
@@ -178,28 +205,46 @@ appgws = {
         }
       }
     }
+    rules = {
+      "http" = {
+        name         = "http"
+        listener_key = "http"
+        backend_key  = "http"
+        rewrite_key  = "xff"
+        priority     = 1
+      }
+    }
   }
 }
 
+# VM-SERIES
 
+ngfw_metrics = {
+  name = "ngwf-log-analytics-wrksp"
+}
 
-# --- VMSERIES PART --- #
-application_insights = {}
-
-vmseries_version = "10.2.3"
-vmseries_vm_size = "Standard_DS3_v2"
-vmss = {
-  "common" = {
-    name              = "common-vmss"
-    vnet_key          = "transit"
-    zones             = ["1", "2", "3"]
-    bootstrap_options = "type=dhcp-client"
-
+scale_sets = {
+  common = {
+    name     = "common-vmss"
+    vnet_key = "transit"
+    image = {
+      version = "10.2.4"
+    }
+    authentication = {
+      disable_password_authentication = false
+    }
+    virtual_machine_scale_set = {
+      bootstrap_options = "type=dhcp-client"
+      zones             = ["1", "2", "3"]
+    }
+    autoscaling_configuration = {
+      default_count = 2
```

Comment:
Shall we put a comment on here stating that this should match with the `default_profile` count in the `autoscaling_profiles` ?

Reply:
Don't really think we should. Both properties are described well in the README and customer can configure all different kinds of autoscaling profiles, ours are just simple examples.